Description: Create a top level makefile
 The original source package does not include a top-level makefile, so we 
 create one.

Author: Daniel Lintott <daniel@serverb.co.uk>

Index: vpcs/Makefile
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ vpcs/Makefile	2013-09-27 12:05:16.658688079 +0100
@@ -0,0 +1,32 @@
+# Makefile for VPCS 0.5b2
+
+# Operating System Type
+# Linux = linux
+# Windows = cygwin
+# MacOSX = osx
+# FreeBSD = fbsd
+export VPCS_OS?=linux
+
+# Executable binary extension
+export PREFIX?=/usr
+export BIN_EXT?=
+
+
+.PHONY: all install clean
+all: vpcs
+
+vpcs:
+	@echo Building VPCS for $(VPCS_OS) on $(ARCH)
+	$(MAKE) -C src -f Makefile.$(VPCS_OS)
+	mv src/vpcs vpcs$(BIN_EXT)
+
+install: vpcs$(BIN_EXT)
+	@echo "Installing VPCS to $(DESTDIR)"
+	install -d $(DESTDIR)$(PREFIX)/bin $(DESTDIR)$(PREFIX)/share/man/man1
+	install vpcs$(BIN_EXT) $(DESTDIR)$(PREFIX)/bin/
+	install -m644 man/vpcs.1 $(DESTDIR)$(PREFIX)/share/man/man1/
+
+clean:
+	$(MAKE) -C src -f Makefile.$(VPCS_OS) clean
+	$(RM) -f vpcs$(BIN_EXT)
+
